artiq/gui/applets.py

```diff
@@ -225,6 +225,7 @@ async def restart(self):
                        "HIST_BIN_BOUNDARIES_DATASET "
                        "HISTS_COUNTS_DATASET"),
     ("Image", "${artiq_applet}image IMG_DATASET"),
+    ("Progress bar", "${artiq_applet}progress_bar VALUE"),
```

Comment:
I added it here, yet it does not appear as a template in the dashboard. Is there another place missing where it should be added?